osu.Game/Configuration/SettingSourceAttribute.cs

```diff
@@ -139,6 +150,30 @@ public static IEnumerable<Drawable> CreateSettingsControls(this object obj)
             }
         }
 
+        public static IEnumerable<(SettingSourceAttribute, PropertyInfo)> GetSettingsSourcePropertiesFromBindables(this object obj)
```

Comment:
Again, needs xmldoc. Probably should rename the `FromAllBindables` method above to `FromBindables` to match this method.